README.md

```diff
@@ -214,6 +214,29 @@ yarn workspace @asap-hub/contentful ctf-migrate create add-foo-field -c bar
 
 Note: the `content_type` is the slug, not the display name, so `externalAuthors` not `External Authors`.
 
+### Generating a migration script from content model
+
+It's possible to make changes in Contentful UI content type, adding or changing fields and then generate the migration.
+
+To do that you need to follow these steps below:
+
+1. Create an empty migration with the command from the section above
+
+   ```sh
+   yarn workspace @asap-hub/contentful ctf-migrate create <name> -c <content_type>
```

Comment:
isn't this the same as https://github.com/yldio/asap-hub/blob/602236c1624ecbc2d8b640526251d98bc9c1ba2b/README.md#L206 ?